src/js/jsx/shared/TextInput.jsx

```diff
@@ -272,8 +272,9 @@ define(function (require, exports, module) {
          * If the value is editable, goes into edit mode
          *
          * @private
+         * @param {bool} select Whether select should be enabled or not
          */
-        _beginEdit: function () {
+        _beginEdit: function (select) {
```

Comment:
It would be better to flip this parameter so that "true" means select is disabled, and to make the boolean parameter optional: `@param {boolean=}`. Then all the current callers don't need to be changed, and the default remains sensible, but it's possible for your special case behavior to override the default.